x-pack/solutions/security/plugins/security_solution/common/api/entity_analytics/entity_store/entities/common.schema.yaml

```diff
@@ -83,30 +135,17 @@ components:
             ingested:
               type: string
               format: date-time
+    
     HostEntity:
       type: object
       required:
-        - host
```

Comment:
There are a bunch of changes to schema that I'm not sure I follow. Can you either drop a comment on what changes or present in Entity Store tech meeting for the sake of better understanding? Not a blocker, just a clarification.